src/Testing/CurlRunnings.hs

```diff
@@ -242,7 +256,7 @@ checkBody state curlCase@(CurlCase _ _ _ _ _ (Just (Exactly expectedValue)) _ _)
                (Just receivedBody)
         else Nothing
 -- | We are checking a list of expected subvalues, and we have a payload to check
-checkBody state curlCase@(CurlCase _ _ _ _ _ (Just (Contains subexprs)) _ _) (Just receivedBody) =
+checkBody state curlCase@(CurlCase _ _ _ _ _ _ (Just (Contains subexprs)) _ _) (Just receivedBody) =
```

Comment:
Note to self: I really need to clean this up with NamedFieldPuns so we don't need to keep adding the `-`'s here.